app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -276,6 +285,107 @@ def parse_namespaces(container_projects)
       parse_base_item(container_projects).except(:namespace)
     end
 
+    def parse_quota(resource_quota)
+      new_result = parse_base_item(resource_quota).except(:namespace)
+      new_result[:container_project] = @data_index.fetch_path(
+        :container_projects,
+        :by_name,
+        resource_quota.metadata["table"][:namespace])
+      new_result[:container_quota_items] = parse_quota_items resource_quota
```

Comment:
@moolitayer any reason for the different style here (not parenthesis)? `parse_quota_items resource_quota` vs `parse_base_item(resource_quota)`